WordPress/Sniffs/PHP/DiscouragedFunctionsSniff.php

```diff
@@ -31,13 +31,6 @@ class WordPress_Sniffs_PHP_DiscouragedFunctionsSniff extends Generic_Sniffs_PHP_
 	 * @var array(string => string|null)
 	 */
 	public $forbiddenFunctions = array(
-		// Deprecated.
```

Comment:
In the code will become very untidy in combination with the upcoming #582 work. I'd rather suggest at some point also re-ordering the `vip` ruleset according to their documentation and making a note in the ruleset about rules like this which are already covered in `core`. Ok ?